templates/vm_mount@.service.j2

```diff
@@ -0,0 +1,19 @@
+# {{ ansible_managed }}
+[Unit]
+Description             = Mounts rootfs for vm %i
+Documentation           = https://github.com/stuvusIT/xen_vman
+BindsTo                 = vm_iscsi@%i.service
+After                   = vm_iscsi@%i.service
+Conflicts               = vm@%i.service
```

Comment:
WHY

Reply:
Same here

Reply:
This is a displaying error on github. Look at the raw file.